java/algorithm/src/test/java/org/apache/arrow/algorithm/sort/TestInsertionSorter.java

```diff
@@ -96,19 +96,22 @@ private void testSortIndicesRange(int start, int end, int vectorLength) {
       InsertionSorter.insertionSort(indices, start, end, comparator);
 
       // verify results
-      for (int i = start, expected = vectorLength - end - 1; i < end; i++, expected++) {
-        assertEquals(expected, indices.get(i));
+      assertEquals(VECTOR_LENGTH, expectedIndices.length);
+      for (int i = 0; i < VECTOR_LENGTH; i++) {
+        assertFalse(indices.isNull(i));
+        assertEquals(expectedIndices[i], indices.get(i));
       }
     }
   }
 
   @Test
   public void testSortIndices() {
-    testSortIntVectorRange(0, 99, 100);
-    testSortIntVectorRange(15, 30, 100);
-    testSortIntVectorRange(50, 50, 100);
-    testSortIntVectorRange(50, 51, 100);
-    testSortIntVectorRange(0, 30, 100);
-    testSortIntVectorRange(72, 99, 100);
+    testSortIndicesRange(2, 5, new int[] {0, 1, 5, 4, 3, 2, 6, 7, 8, 9});
+    testSortIndicesRange(3, 7, new int[] {0, 1, 2, 7, 6, 5, 4, 3, 8, 9});
+    testSortIndicesRange(3,4, new int[] {0, 1, 2, 4, 3, 5, 6, 7, 8, 9});
```

Comment:
nit: spacing between 1rst to args.

Reply:
Resolved. Thank you. Maybe we need to further enhance Java style checking.